### PR TITLE
feat(tutorial): Several updates

### PR DESCRIPTION
**Pull request by qbzzt**
1. `README.md`: Make more readable and easier to use.
2. `.env.example`: Made clearer
3. `package.json`: Added `--legacy` so it'll work with a mainnet fork.
4. Updated the FAQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "anvil": "source .env && anvil --fork-url $ANVIL_FORK_URL --chain-id $ANVIL_CHAIN_ID",
     "deploy": "source .env && forge create contracts/src/Counter.sol:Counter --rpc-url $FORGE_RPC_URL --private-key $FORGE_PRIVATE_KEY --etherscan-api-key $ETHERSCAN_API_KEY --verify",
-    "deploy:anvil": "source .env && forge create contracts/src/Counter.sol:Counter --rpc-url http://127.0.0.1:8545 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
+    "deploy:anvil": "source .env && forge create contracts/src/Counter.sol:Counter --rpc-url http://127.0.0.1:8545 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 --legacy",
```
Comment (roninjin10) [inline package.json:8]: why is this necessary?   Because it's not bedrock?

**Review by roninjin10 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (qbzzt) [reply package.json:8]: Exactly. `--legacy` is not required for Goerli, but it is required for mainnet.
